Add initial CLI docs (#414)
saleor/saleor-docs: docs/cli.mdx

       ▄███▀               ░░░░░░░░░░░░░░░░░░▄███▀░░░░░░░░░░░░
    ▄███▀                                 ▄███▀
 ▄█████████████████████████████████████████▀         Saleor Commerce CLI v0.21.0

Usage: saleor <command> [options]

Commands:
  saleor info                    Hello from Saleor
  saleor login                   Log in to the Saleor Cloud
  saleor configure [token]       Configure Saleor CLI
  saleor register                Create Saleor account
  saleor organization [command]                                   [aliases: org]
  saleor environment [command]                                    [aliases: env]
  saleor backup [command]
  saleor job [command]
  saleor project [command]
  saleor storefront [command]                                   [aliases: store]
  saleor telemetry [command]                                     [aliases: tele]
  saleor webhook [command]                                       [aliases: hook]
  saleor app [command]
  saleor vercel [command]

Options:
  -V, --version  Show version number                                   [boolean]
  -h, --help     Show help                                             [boolean]

> for more information, find the documentation at https://saleor.io

Reviewer: Documentation is at docs.saleor.io

Author: @dekoza done

---
title: Saleor CLI 
---

> ## Saleor CLI

Reviewer 2: <img width="1164" alt="image" src="https://user-images.githubusercontent.com/5188636/166684619-fb2bd172-19cc-4fba-95c7-b2a68b61ec3a.png">

I believe this header can be skipped because it's a duplication of the page title.

Saleor provides a convenient command-line tool to speed the development around Saleor. Using our CLI you can interact with the Saleor platform using a terminal or through an automated system to quickly create and manage Saleor environments, test storefront installation and deploy to cloud providers such as Vercel or Netlify, and more.

This page contains a complete list of all available Saleor CLI commands with their parameters. 

> ### Installing the CLI

Reviewer 2: ```suggestion
## Installing the CLI
```

Reviewer 2: I believe levels of all headers could be one level higher

Type `saleor organization` with no additional parameters to see a list of available sub-commands:

```
saleor organization [command]

Commands:
  saleor org show [slug]         Show a specific organization
  saleor org list                List organizations
  saleor org create [name]       Create a new organization
  saleor org remove [slug]       Remove the organization
  saleor org permissions [slug]  List organization permissions
  saleor org switch [slug]       Make the provided organization the default one

Options:
  -V, --version  Show version number                                   [boolean]
  -h, --help     Show help
```

#### environment

The `saleor environment` command allows you to manage environments in your Saleor Cloud account. You can use it to list available environments, create new ones, remove existing ones, upgrade the Saleor engine to a specific version or manage the database for a particular environment.

Type `saleor environment` with no additional parameters to see a list of available sub-commands:

```
saleor environment [command]

Commands:

>   saleor env show [environment]      Show a specific environmet

Reviewer 2: ```suggestion
  saleor env show [environment]      Show a specific environment
```

  saleor env list                    List environments

>   saleor env create [name]           Create a new environmet

Reviewer 2: ```suggestion
  saleor env create [name]           Create a new environment
```

>   saleor env switch [key]            Make the provided environmet the default on
>                                      e

Reviewer 2: ```suggestion
  saleor env switch [key]            Make the provided environment the default on
```

Seems like a cut of description?

```
saleor webhook [command]

Commands:
  saleor webhook list    List webhooks for an environment
  saleor webhook create  Create a new backup
  saleor webhook edit    Edit a webhook
  saleor webhook update  Update webhooks for an environment

Options:
  -V, --version  Show version number                                   [boolean]
  -h, --help     Show help                                             [boolean]
```

#### app

The `saleor app` command allows you to manage Saleor apps. You can use it to list available Saleor apps, create a project structure for a new Saleor app or expose it remotely with our built-in tunneling. 

Type `saleor app` with no additional parameters to see a list of available sub-commands:

```
saleor app [command]

Commands:
  saleor app list           List webhooks for an environment
  saleor app install        Install a Saleor App by URL
  saleor app tunnel [port]  Expose your Saleor app remotely via tunnel
  saleor app create [name]  Create a Saleor App template

Options:
  -V, --version  Show version number                                   [boolean]
  -h, --help     Show help                                             [boolean]

```

> #### vercel

Reviewer 2: Missing description
